media/css/firefox/whatsnew/whatsnew-lite.scss

```diff
@@ -63,158 +63,161 @@
 //* -------------------------------------------------------------------------- */
 // Page content
 
-.lite-container {
-    display: flex;
-    justify-content: flex-end;
-    align-items: center;
-    flex-direction: column;
-    margin-top: $spacing-md;
+.lite-body {
+    background-color: #89eae3;
+    background-image: url('/media/img/firefox/whatsnew/lite/background.jpg');
+    background-position: top right;
+    background-size: auto 100%;
+    background-repeat: no-repeat;
+    color: $color-gray-80;
+    position: relative;
+    @include font-firefox;
+    @media #{$mq-md} {
+        background-size: 150% auto;
+        background-position: top left;
+    }
+}
 
+.lite-content {
+    max-width: 1600px;
+    padding-left: $spacing-lg;
+    padding-right: $spacing-lg;
+    background-repeat: no-repeat;
+    background-position: top left;
+    background-size: 42%;
     @media #{$mq-md} {
-        min-height: 180px;
+        display: flex;
+        justify-content: flex-end;
+        background-image: url('/media/img/firefox/whatsnew/lite/photo.webp');
```

Comment:
webp! 🔥

Reply:
@pmac I think this is the first .webp image format we've added to bedrock. Just checking, is there anything we need to do to make sure this is served correctly via the CDN? I assume this should be fine, but thought I should double-check.

Reply:
Hmmm... Only issue I could foresee would be that it would be served with the wrong content-type header. I'll check to see if our stuff should understand webp.

Reply:
A quick note on the rationale for using WebP here: 
* This page is aimed at recent versions of  Firefox only, so support isn't an issue
* The design called for an unusual combination of a circularly-masked photographic element (usually best compressed via JPEG) and simpler solid color elements.

Originally I was going to either embed a JPEG (for the photographic element) in an SVG (for the masking and simpler shapes), or put a JPEG as a background element on a div with border-radius to handle the masking.

WebP was much simpler than both options. I used the cleverly-named [WebPShop](https://developers.google.com/speed/webp/docs/webpshop) plugin for Photoshop to export the WebP image

Reply:
It does seem that our mime-type guessing probably won't get the right answer for the webp images. It will probably do `binary/octet-stream`. I think the image will still work on the page, but not if viewed directly. Thoughts?

Reply:
If we push this to demo, we should be able to test that it works ok, right?

Reply:
I think demos serve the static media themselves, where dev, stage, and prod sync the media to an s3 bucket and serve from there. I don't think we'll be sure until we get it on dev unfortunately :/

Reply:
Ok, let's see how this goes once it merges. Is it worth filing a separate issue so that we can fix the mime-type?

Reply:
I think so. We should really send the proper mime type. Let's verify that it is in fact the wrong type, then we can file an issue to fix it.

Reply:
I'm going to go ahead and file this issue. I'm pretty confident it will be wrong, but I've found a pretty easy way to fix it, so I'll document that in the issue. Good catch!